docs/pages/guides/building-for-tv.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Building for TV
+description: A guide for building an Expo app for an Apple TV or Android TV target.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+> **Warning** TV development is available only in SDK 50 and later, and is still considered experimental. Not all Expo features and SDK modules are available on TV.
+
+React Native is supported on Apple TV and Android TV through the [React Native TV repo](https://github.com/react-native-tvos/react-native-tvos).  It is not just for TV — it is a full fork of the core repo, and supports both phone and TV targets, including Hermes and Fabric support
+
+By taking an Expo project and using the React Native TV repo as the `react-native` dependency, the project can be made to target either phone (Android, iOS) or TV (Android TV, Apple TV) devices.
+
+The changes needed to the native Android and iOS files are minimal, and are made with a [config plugin](https://github.com/react-native-tvos/config-tv/tree/main/packages/config-tv):
+
+- iOS:
+  - **ios/Podfile** is modified to target tvOS instead of iOS
+  - The Xcode project is modified to target tvOS instead of iOS
+  - The splash screen (**SplashScreen.storyboard**) is modified to work on tvOS
+- Android:
+  - **AndroidManifest.xml** is modified
+  - The default phone portrait orientation is removed
+  - The required intent for TV apps is added
+  - **MainApplication.kt** is modified to remove unsupported Flipper invocations
+
+## Quick start
+
+The fastest way to generate a new project is described in the [TV Example](https://github.com/expo/examples/tree/master/with-tv) in the Expo examples repo.
+
+## Step by step
+
+If you already have an existing Expo project, this walkthrough shows what is required to take an Expo project and modify it for TV.
+
+You will need
+
+- An Expo project using SDK 50 or later.
+- Apple TV:
+  - Xcode 15 or later
+  - tvOS SDK 17 or later. (This is not installed automatically with Xcode. If you did not install it at the prompt when first opening Xcode, you can install later with `xcodebuild -downloadAllPlatforms`).
+- Android TV:
+  - In the Android Studio SDK manager, select the dropdown for the Android SDK you are using (API version 31 or later), and make sure an Android TV system image is selected for installation. (For M1 Macs, choose the ARM 64 image, otherwise, choose the Intel x86_64 image).
```

Comment:
```suggestion
  - In the Android Studio SDK manager, select the dropdown for the Android SDK you are using (API version 31 or later), and make sure an Android TV system image is selected for installation. (For Apple Silicon Macs, choose the ARM 64 image, otherwise, choose the Intel x86_64 image).
```